ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/type-component/multischema-type.component.ts

```diff
@@ -31,9 +33,31 @@ import { set } from 'lodash';
   `,
   styleUrls: ['./multischema-type.component.scss'],
 })
-export class GioFjsMultiSchemaTypeComponent extends FieldType implements OnInit {
+export class GioFjsMultiSchemaTypeComponent extends FieldType implements OnInit, OnDestroy {
+  private unsubscribe$: Subject<void> = new Subject<void>();
+
+  constructor(private readonly cdr: ChangeDetectorRef) {
+    super();
+  }
+
+  public ngOnDestroy() {
+    this.unsubscribe$.next();
+    this.unsubscribe$.unsubscribe();
+  }
+
   public ngOnInit(): void {
     set(this.field, 'fieldGroup[0].props.appearance', 'fill');
     set(this.field, 'fieldGroup[0].props.label', 'Select option');
+    set(this.field, 'fieldGroup[0].props.disabled', this.form.disabled);
+
+    this.form.valueChanges
```

Comment:
Here I use the `form.valueChanges` because the following code block is not enough to disable the field group

```js
this.field?.options?.fieldChanges?.pipe(
        tap(() => {
          set(this.field, 'fieldGroup[0].props.disabled', this.form.disabled);
          this.cdr.detectChanges();
        }),
        takeUntil(this.unsubscribe$),
    ).subscribe();
```